nav-app/src/assets/config.json

```diff
@@ -244,6 +244,9 @@
             {"name": "link_underline", "enabled": true, "description": "Disable to remove the hyperlink underline effect on techniques."},
             {"name": "metadata", "enabled": true, "description": "Disable to remove the ability to add metadata to techniques."},
             {"name": "clear_annotations", "enabled": true, "description": "Disable to remove the button to clear all annotations on the selected techniques."}
+        ]},
+        {"name": "toolbar_controls", "enabled": true, "description": "Disable to disable all subfeatures", "subfeatures": [
+            {"name": "sticky_toolbar", "enabled": true, "description": "Disable to remove the sticky toolbar."}
```

Comment:
I'd recommend updating the description of "sticky_toolbar" to something like: "Disable to remove the ability to enable/disable the sticky toolbar". The current description implies that the sticky toolbar will be disabled s.t. it will not remain at the top of the screen on scroll, if a user changes this to `"enabled": false`. Disabling this feature will only remove the icon from the toolbar, but the sticky toolbar will still be enabled by default.